### PR TITLE
added skipping auto enrollment info

### PR DESCRIPTION
**Pull request by Malind19**

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/windows-autopilot/enrollment-status.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ ms.topic: article
 The Windows Autopilot Enrollment Status page displaying the status of the complete device configuration process.  Incorporating feedback from customers, this provides information to the user to show that the device is being set up and can be configured to prevent access to the desktop until the configuration is complete. 
  
  ![Enrollment status page](images/enrollment-status-page.png)
+ 
+From Windows 10 version 1803 onwards, you can opt-out of the account setup phase. When it is skipped, the settings will be applied for the users when as they access their desktop for the first time.
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/enrollment-status.md]:
```suggestion
From Windows 10 version 1803 and onwards, you can opt out of the account setup phase. If it is skipped, settings will be applied for users when they access their desktop for the first time. 
```

**Comment by mypil** [comment]
@greg-lindsay - Can you please assist to check, approve and merge the content for this PR based on issue #3263?

Thank you.

**Comment by mypil** [comment]
@greg-lindsay - I would like to follow up. Can you please assist to check, approve and merge the content for this PR based on issue #3263?

Thank you.

**Comment by mypil** [comment]
@greg-lindsay - We would like to follow up. Can you please assist to check, approve and merge the content of this PR?

Thank you.

**Pull request merged**